### PR TITLE
Avoid creating temporary files in wget test

### PR DESCRIPTION
**Pull request by rjoleary**
Signed-off-by: Ryan O'Leary <ryanoleary@google.com>

### CODE REVIEW

**Comment by rjoleary** [comment]
For #1955

File: cmds/core/wget/wget_test.go [hunk 1]
```diff
@@ -40,55 +41,56 @@ func (h handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
...
 	}, {
-		// ipv4
+		name:    "ipv4",
 		flags:   []string{},
 		url:     "http://127.0.0.1:%[1]d/200",
 		content: content,
 		retCode: 0,
-	}, /*{ TODO: travis does not support ipv6
-		// ipv6
+	}, /*{ CircleCI does not support ipv6
```
Comment (rminnich) [inline cmds/core/wget/wget_test.go:62]: could you just use a // comment here for lines 62-67? I find this as written a bit hard to parse..

**Review by rminnich — DISMISSED** [review]

**Review by rminnich — APPROVED** [review]
